### PR TITLE
Feature/draw cards

### PR DESCRIPTION
**Pull request by psk44**
I wrote a function that will draw shuffled cards until there is none left in the deck.

### CODE REVIEW

File: source/CardGames.py [hunk 1]
```diff
@@ -77,6 +81,20 @@ def getCard(self):
     self.discarded.append(card)
     return card
```
Comment (kerenvr) [inline source/CardGames.py:85]: #your function

File: source/CardGames.py [hunk 2]
```diff
@@ -77,6 +81,20 @@ def getCard(self):
...
+        Card: The card drawn from the top of the deck.
+      """
+      if self.size > 0:
+        card = self.getCard()
+        print(f"Card drawn: {card.suit} {card.value}")
+        return card
+      else:
+        print("No cards left in the deck.")
+        return None
+
```
Comment (kerenvr) [inline source/CardGames.py:99]: This looks good. Now, all we need to do is put this option on the starter screen and figure out a way for users to call this function. Maybe we can bind it to a key somehow, or make it to where they can simply call it in the terminal (without the class.drawCard()).

**Review by kerenvr — APPROVED** [review]
The function is good to go for the draw card option when someone has to go fish. Now we just need to include it in our go fish function and make it to where users can easily call it.

**Comment by TommyMedina** [comment]
LOOK OUT!!!
I was just looking around and noticed that this branch is set to merge with main. I thought I should let you know as this is probably not what was intended.

File: source/CardGames.py [hunk 3]
```diff
@@ -77,6 +81,20 @@ def getCard(self):
...
+      Returns:
+        Card: The card drawn from the top of the deck.
+      """
+      if self.size > 0:
+        card = self.getCard()
+        print(f"Card drawn: {card.suit} {card.value}")
+        return card
+      else:
+        print("No cards left in the deck.")
+        return None
```
Comment (RobertWheel) [inline source/CardGames.py:98]: looks all good!

**Pull request merged**